README.md

```diff
@@ -194,6 +194,26 @@ abstract class CounterBase with Store {
 }
 ```
 
+#### Asynchronous Actions
+
+MobX.dart handles asynchronous actions [automatically](https://github.com/mobxjs/mobx.dart/pull/89) and does not require wrapping the code with ```runInAction```. 
```

Comment:
I think we should skip the part about mentioning the pull request. Too much detail for our users ;-). Also the `runInAction` should link to that section on API. There is no need to use the triple quoted string. Instead a plain backquoted string should be good enough, such as \`runInAction\`

Reply:
Thanks. I updated it but for some reason cannot see the changes in the PR.

Reply:
I see them now.